src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -216,6 +216,10 @@ private function normalizeDefaults(array $defaults): array
             }
         }
 
+        if (!\array_key_exists('stateless', $defaults)) {
+            @trigger_error("The node 'api_platform.defaults.stateless' should be specified", E_USER_DEPRECATED);
```

Comment:
```suggestion
        if (true !== $defaults['stateless'] ?? null) {
            @trigger_error('Not setting the "api_platform.defaults.stateless" configuration option to `true` is deprecated since API Platform 2.6 and it will default to `true` in 3.0. You can override this at the operation level if you have stateful operations (highly not recommended).', E_USER_DEPRECATED);
```

Reply:
Yes, much better!
But I may recommend to write:
> Not setting the "api_platform.defaults.stateless" configuration option to `true` is deprecated since API Platform 2.6 and it will default to `true` in 3.0. You can override this **at the resource or the operation level** if you have stateful operations (highly not recommended).

instead.
WDYT ?

Reply:
While technically correct, I don't want to bring attention to that because what's a stateful resource?

Reply:
Yes you're right